### PR TITLE
Adapt format of stanford open ie

### PR DESCRIPTION
**Pull request by simonzachau**

### CODE REVIEW

File: algorithms/openie-stanford/relationship.js [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
 const filter = require('./relationship.filter.js');
 
 const client = new StanfordCoreNLPClient(
   undefined,
-  'tokenize, ssplit, pos, depparse, relation, openie, ner'
+  'tokenize, ssplit, pos, depparse, relation, openie'
 );
 
-var data = '';
+let data = '';
```
Comment (Henni) [inline algorithms/openie-stanford/relationship.js]: why is this variable global?

File: app/algorithms.js [hunk 2]
```diff
@@ -36,110 +38,129 @@ module.exports.call = function(websites) {
...
+            } else {
+              // write to db
+              console.log('OpenIE W: Write JSON to DB');
+              dbConnection.writeRelationships(result);
+            }
+          } else {
+            console.log('OpenIE W: Finished, but result was ' + result);
+          }
+        }, error => {
+          console.error('OpenIE W: ' + error);
```
Comment (Henni) [inline app/algorithms.js:122]: we have a lot of code duplication here. @Sandr00 and I are working on removing it in a separate PR.

**Review by Henni — CHANGES_REQUESTED** [review]
Otherwise looks good!

File: algorithms/openie-stanford/config.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "core_nlp_server_url": "http://nlp-openie-stanford.azurewebsites.net",
+  "core_nlp_server_url": "http://13.79.167.74",
```
Comment (Sandr0x00) [inline algorithms/openie-stanford/config.json:2]: Should we really have the url here? I think env variables should be there as well.

File: app/algorithms.js (hunk 2 above)
Comment (Sandr0x00) [reply app/algorithms.js:122]: When this PR is in ;)

**Review by Henni — APPROVED** [review]

**Pull request merged**